project.clj

```diff
@@ -65,7 +65,8 @@
               ["dev"]
 
               :injections [(require 'io.aviso.logging.setup)]
-              :dependencies [[io.aviso/logging "0.3.1"]
+              :dependencies [[aero "1.1.3"]
```

Comment:
Can we use `environ` instead? Aero encourages a "two config file" pattern which is a bit verbose.

Reply:
How does aero encourage a two config file pattern?

    When config is left out of source code control it festers
    and diverges from the code base. Better to keep a single 
    config file in source code control.    

-- https://github.com/juxt/aero#allow-config-to-be-stored-in-the-source-code-repository-